src/Lumper.UI/Views/Bsp/Lumps/PakFile/PakFileLumpView.axaml.cs

```diff
@@ -47,13 +47,17 @@ public IClassicDesktopStyleApplicationLifetime Desktop
         if (result is not { Count: 1 })
             return path;
 
-        if (result[0].TryGetUri(out var uri))
+        try
         {
-            path = uri.AbsolutePath;
+            return result[0].Path.AbsolutePath;
+        }
+        catch (InvalidOperationException)
+        {
+            return null;
```

Comment:
Sorry, I don't remember exactly what but there were some changes to IStorageFile in Avalonia 11 that I'm pretty sure *were* affecting the compile here, which was the point of the commit.

And sorry, I probably should've just dropped this entire commit since so the majority of the UI work got rewritten after this and I committed separately. Since we're here discussing it it's probably easier to just ignore this commit rather than do a force-push that changes the history so drastically.